rero_ils/modules/persons/listener.py

```diff
@@ -30,4 +30,5 @@ def enrich_persons_data(sender, json=None, record=None, index=None,
     :param doc_type: The doc_type for the record.
     """
     if index == '-'.join([PersonsSearch.Meta.index, doc_type]):
-        json['organisations'] = record.organisation_pids
+        person = Person.get_record_by_pid(record.get('pid'))
```

Comment:
is it really required?

Reply:
Yes, because in the next line `json['organisations'] = person.organisation_pids` the `organisation_pids` is a `property` of the `Persons` class.